### DISCUSSION TITLE
Leading Zeroes in the Day

### DISCUSSION THREAD

**Discussion by sbazzle** [totaldebug/atomic-calendar-revive#1498] [Feature Requests]
I installed Atomic Calendar into my dashboard a couple weeks ago, and now that we're approaching the first day of the upcoming month, I noticed that the day of the month has a leading zero, i.e. "Sat 01" in my calendar below.  I'm personally not a fan of the leading zeroes in days, and I'm just wondering if there's a method of not showing them, i.e. "Sat 1".

![Capture](https://github.com/totaldebug/atomic-calendar-revive/assets/48576150/4ea06d44-b14b-4662-82fd-8c0f26ff4c79)

**Comment by marksie1988** [comment]
This will be possible to change in the latest version that i plan to release shortly.
